hack/update-codecgen.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+cd "${KUBE_ROOT}"
+
+generated_files=`
+  find . -not \( \
+      \( \
+        -wholename './output' \
+        -o -wholename './_output' \
+        -o -wholename './release' \
+        -o -wholename './target' \
+        -o -wholename '*/third_party/*' \
+        -o -wholename '*/Godeps/*' \
+      \) -prune \
+    \) -name '*.generated.go'`
+
+for generated_file in ${generated_files}; do
+	initial_dir="${PWD}"
+  file=`echo "$generated_file" | sed 's/.generated.go/.go/g'`
+  # codecgen work only if invoked from directory where the file
+  # is located.
+  cd `dirname "${file}"`
+  base_file=`basename "$file"`
+  base_generated_file=`basename "$generated_file"`
+  codecgen -d 1234 -o "${base_generated_file}" "${base_file}"
```

Comment:
Can you use a variable or at least comment on `-d 1234` so people won't have to go look it up like me.